.github/workflows/ci-e2e.yaml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - main
       - "release-*"
+      - separate-policy-controller
```

Comment:
@maleck13 Can be removed. The e2e works, but the PR one will always fail until this is merged since its using the workflow from main.